test/unit/JSONAPISerializer.test.js

```diff
@@ -2154,5 +2155,107 @@ describe('JSONAPISerializer', function() {
     expect(converted['array-of-number']).to.deep.equal([1, 2, 3, 4, 5]);
     expect(converted.date).to.be.a('Date');
   });
- }); 
-});
+ });
+ 
+ describe('LRU Cache', function() {
```

Comment:
LRU cache tests should be in a separate file.

Reply:
Should be all set